gcloud-java-resourcemanager/src/main/java/com/google/gcloud/spi/ResourceManagerRpc.java

```diff
@@ -121,5 +123,27 @@ public Y y() {
    */
   Project replace(Project project);
 
+  /**
+   * Returns the IAM policy associated with a project.
+   *
+   * @throws ResourceManagerException upon failure
+   */
+  Policy getPolicy(String projectId);
+
+  /**
+   * Replaces the IAM policy associated with the given project ID.
```

Comment:
Nit: in `getPolicy` we say "associated with a project" while here ewe say "with the given project ID". I think we should either refer to the project or the project ID. My preference is for removing ID.